website/docs/guides/core-to-cloud-1.md

```diff
@@ -0,0 +1,244 @@
+
+dbt Core is an open-source tool that enables data teams to transform data following analytics engineering best practices using a command line interface. It must be self-hosted and maintained.
+
+### What you'll learn
+
+This guide outlines the steps you need to take to move from dbt Core to dbt Cloud and highlights the necessary technical changes:
+
+- [Account setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=3#account-setup): Learn how to create a dbt Cloud account, invite team members, and configure it for your team.
+- [Environment variables](https://docs.getdbt.com/guides/core-to-cloud-1?step=4#environment-variables): Discover how to manage environment variables in dbt Cloud, including their priority.
+- [Data platform setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=5#data-platform-setup): Find out about connecting your data platform to dbt Cloud.
+- [Git setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=6#git-setup): Learn to link your dbt project's Git repository with dbt Cloud.
+- [Developer setup:](https://docs.getdbt.com/guides/core-to-cloud-1?step=7#developer-setup) Understand the setup needed for developing in dbt Cloud.
+- [Orchestration setup](https://docs.getdbt.com/guides/core-to-cloud-1?step=8#orchestration-setup): Learn how to prepare your dbt Cloud environment and jobs for orchestration.
+- [Models configuration](https://docs.getdbt.com/guides/core-to-cloud-1?step=9#models-configuration): Get insights on validating and running your models in dbt Cloud, using either the dbt Cloud IDE or dbt Cloud CLI.
+- [What's next?](https://docs.getdbt.com/guides/core-to-cloud-1?step=10#whats-next): Summarizes key takeaways and introduces what to expect in the following guides.
+
+## Prerequisites
+
+- You have an existing dbt Core project connected to a data platform and Git repository.
+- You have a dbt Cloud account. **[Don't have one? Start your free trial today](https://www.getdbt.com/signup)**!
+
+### Related docs
+- [Learn dbt Cloud](https://courses.getdbt.com/collections)
+- [Develop with dbt Cloud](/docs/cloud/about-develop-dbt)
+- [Deploy jobs](/docs/deploy/deployments)
+- Book [expert-led demos](https://www.getdbt.com/resources/dbt-cloud-demos-with-experts) and insights
+- Work with the [dbt Labs’ Professional Services](https://www.getdbt.com/dbt-labs/services) team to support your data organization and migration.
+
+## Account setup
+
+1. [Create your dbt Cloud account](https://www.getdbt.com/signup).
+
+2. Learn about [access](/docs/cloud/manage-access/about-user-access) and [invite users](/docs/cloud/manage-access/about-user-access) to your dbt Cloud account and dbt Cloud project.
+
+3. Configure [Single Sign-On (SSO)](/docs/cloud/manage-access/sso-overview) or [Role-based access control (RBAC)](/docs/cloud/manage-access/about-user-access#role-based-access-control) for easy and secure access. <Lifecycle status='enterprise' />
```

Comment:
Somewhere (maybe here, maybe elsewhere) we should call out that these are Enterprise features and if they remain on a Team or Developer plan after the trial period, they won't have access to some of these features.

Reply:
moved this over to slack